.github/workflows/azure-dev.yml

```diff
@@ -12,8 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      # Pinned to 1.3.0 while we await feedback on an issue that surfaces after upgrading Bicep starting in version 1.3.1
-      image: mcr.microsoft.com/azure-dev-cli-apps:1.3.0
+      image: mcr.microsoft.com/azure-dev-cli-apps:latest
```

Comment:
Should we lock it to a specific version?  Given the experience with 1.3.1 upgrade, I think locking it to something that works in the Dev Container is a good idea.